web/themes/custom/oc_bootstrap/css/style.css

```diff
@@ -976,17 +976,56 @@ article.teaser h2 a {
 }
 
 .page-node-type-job h1.page-header > span {
-  border: none;
+  border: 2px solid;
+  width: 100%;
+}
+
+.page-node-type-job h1.page-header {
+  width: 100%;
+  margin-left: auto;
+  left: 0;
```

Comment:
do we need the left 0?

Reply:
So, without the left 0 or margin left, we get this result:
 
![Screenshot 2021-03-16 at 13 58 32 (2)](https://user-images.githubusercontent.com/58986635/111321835-363ecd80-8660-11eb-8b81-4203e414fbe1.png)

Reply:
Oh, I see, because its overriding the left 50% from normal headers.